source/php/Component/Hero/hero.blade.php

```diff
@@ -7,9 +7,9 @@
 
     @if ($title || $paragraph || $byline)
         <div class="o-container {{ $baseClass }}__container">
-            @if($videoUrl)
+            @if($video)
```

Comment:
This will show video if video === "". A solution would be @if(!empty($video)). Maybe this is not needed, fix or keep then merge!

Reply:
Stronger url validation will be done in implementation phase (modularity).